### PR TITLE
Add the silde of meetup 40 and 41

### PR DESCRIPTION
**Pull request by ydFu**
* Add the sildes in meetup 40 and 41.
* Add the OCF Donate link in README.md .
Signed-off-by: ydFu <ader.ydfu@gmail.com>

**Before submitting a pull request**, please make sure the following is done:

### Checklist

- [x] Fork [the repository](https://github.com/cloud-native-taiwan/meetups) and create your new branch from `master`.
- [x] Please mention the name of community in Pull Request title.
- [x] Git Commit Messages
    - [x] Use the present tense (Also in Pull Request title): "Add feature" not "Added feature"
    - [x] Use the imperative mood (Also in Pull Request title): "Move cursor to..." not "Moves cursor to..."
    - [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ Cloud Native Computing Taiwan User Group is dedicated to combine community of Cl
 - [KKTIX](https://cntug.kktix.cc)
 - [HackMD](https://hackmd.io/@CNTUG)
 - [Meetup 投影片(Slides)](slides/README.md)
-- Cooperative communities.
+- [OCF Donate](https://ocf.neticrm.tw/civicrm/contribute/transact?reset=1&id=29&_ga=2.247282817.443417200.1638513519-757737249.1637652904)
```
Comment (iGene) [inline README.md:13]: IMO, it's better to only leave id=29 and delete other arguments to make it shorter.

**Review by iGene — COMMENTED** [review]
Small comment

File: README.md (hunk 1 above)
Comment (ydFu) [reply README.md:13]:
```suggestion
- [OCF Donate](https://ocf.neticrm.tw/civicrm/contribute/transact?id=29)
```
Thank you for your carefulness.
I also agree the query string to leave only what is necessary.
Is the above adjustment content better?

**Review by iGene — APPROVED** [review]
LGTM

**Review by kairen — APPROVED** [review]

**Pull request merged**